flight/Modules/Stabilization/stabilization.c

```diff
@@ -105,6 +108,11 @@ enum {
 	PID_MAX
 };
 
+// Check if the axis enumeration maps properly to the things above.
+DONT_BUILD_IF((PID_RATE_ROLL+0 != ROLL)||(PID_ATT_ROLL != PID_GROUP_ATT+ROLL), stabAxisPidEnumMapping1);
```

Comment:
Can't compare anonymous enums without the compiler throwing a fit about it. +0 is shorter than a cast to fix this.